### PR TITLE
ui: Add Data Usage information

### PR DESCRIPTION
**Pull request by connected-adietz**
- new Data Usage information shown on the back of the main card, accessed by clicking info icon in top left corner which flips the card to the back
- Download and upload speeds are shown on graph on Data Usage card, and refreshed every second
- new FlipCard component added for flipping animation
- new NetworkSpeedVisual component added for graph creation and display

### CODE REVIEW

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://639-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://641-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 2</h3>
    Details: <strong><a href='https://652-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://653-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://655-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://656-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

File: ui/src/UI/Components/Card/FlipCard.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            FlipCard f = sender as FlipCard;
+            if (f != null)
+            {
+                f.OnPropertyChanged("IPAddress");
+            }
+        }
+
+        private void UpdateIPAddressString()
+        {
+            IPAddressString = string.Empty;
```
Comment (connected-nkozul) [inline ui/src/UI/Components/Card/FlipCard.xaml.cs]: No need.

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://657-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

File: ui/src/WireGuard/Connector.cs [hunk 2]
```diff
@@ -36,6 +36,10 @@ public static bool Connect(bool switchServer = false, string previousServerCity
 
             if (switchServer && Manager.MainWindowViewModel.Status == Models.ConnectionState.Protected)
             {
+                // Updae IP info.
```
Comment (connected-nkozul) [inline ui/src/WireGuard/Connector.cs]: Typo

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://677-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Comment by mhuang-connectedqa** [comment]
<h3>Errors: 0, Warnings: 0</h3>
    Details: <strong><a href='https://687-220557697-gh.circle-artifacts.com/0/test/result/Output.log' target='_blank'>details</a></strong>

**Pull request merged**